src/client/react/components/CustomFunctionsDashboard/Summary/Summary.tsx

```diff
@@ -1,21 +1,33 @@
 import * as React from 'react';
 import styled from 'styled-components';
-
```

Comment:
nit: i personally prefer to organize imports like the following:

most necessary third party imports

first party imports